### PR TITLE
Fix: missing change event trigger

### PR DESCRIPTION
**Pull request by admwx7**
Resolves issue #38

### CODE REVIEW

**Comment by notwaldorf** [comment]
This looks great thanks! Before I merge it, would you mind adding a test that this event actually fires, please? 🙏

**Comment by admwx7** [comment]
@notwaldorf I had to change the logic once a test was in place, looks like it's a different issue from paper-input somehow but when we have a shadowRoot it gets retargetted and propagated for us, but when shadowRoot isn't there I have to manually create a new event, that work for you?

**Comment by notwaldorf** [comment]
Looking at https://github.com/PolymerElements/paper-input/blob/master/paper-input-behavior.html#L519, don't you also want bubbles and canceleable?

File: gold-zip-input.html [hunk 1]
```diff
@@ -140,8 +141,20 @@
...
             this.$.input.selectionEnd = start + 1;
           }
-        }
+        },
 
+        /**
+         * Recreates a change event that's targetted against this element if it
+         *  doesn't have a shadowRoot, turns out that's handled for us if it does.
+         */
+        __onChange: function() {
```
Comment (notwaldorf) [inline gold-zip-input.html]: Actually, why do you need this at all? Doesn't it come from PaperInputBehavior, where it's already implemented?

**Comment by admwx7** [comment]
Rebased and re-opened in https://github.com/PolymerElements/gold-zip-input/pull/45

**Pull request closed without merging**